From: hyoyeong <164199371@qq.com>
Date: Thu, 27 Aug 2026 02:32:09 +0000
Subject: Translated using Weblate (Korean)

Currently translated at 97.8% (89 of 91 strings)

Translation: openkylin-huanghe/ukui-search
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukui-search/ko/
---
 translations/ukui-search/ko.ts | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/translations/ukui-search/ko.ts b/translations/ukui-search/ko.ts
index d24aa70..7c9e33d 100644
--- a/translations/ukui-search/ko.ts
+++ b/translations/ukui-search/ko.ts
@@ -62,7 +62,7 @@
         <location filename="../../libsearch/aisearch/ai-search-plugin.cpp" line="148"/>
         <location filename="../../libsearch/aisearch/ai-search-plugin.cpp" line="352"/>
         <source>Can not get a default application for opening %1.</source>
-        <translation>%1을(를) 여는 기본 응용 프로그램을 가져올 수 없습니다.</translation>
+        <translation>%1을(를) 여는 기본 응용 프로그람을 가져올 수 없습니다.</translation>
     </message>
     <message>
         <location filename="../../libsearch/aisearch/ai-search-plugin.cpp" line="167"/>
@@ -137,7 +137,7 @@
         <location filename="../../libsearch/appsearch/app-search-plugin.cpp" line="32"/>
         <location filename="../../libsearch/appsearch/app-search-plugin.cpp" line="250"/>
         <source>Add Shortcut to Desktop</source>
-        <translation>바탕화면에 바로가기 추가</translation>
+        <translation>바탕 화면에 바로가기 추가</translation>
     </message>
     <message>
         <location filename="../../libsearch/appsearch/app-search-plugin.cpp" line="33"/>
@@ -155,17 +155,17 @@
         <location filename="../../libsearch/appsearch/app-search-plugin.cpp" line="64"/>
         <location filename="../../libsearch/appsearch/app-search-plugin.cpp" line="69"/>
         <source>Applications Search</source>
-        <translation>응용 프로그램 검색</translation>
+        <translation>응용 프로그람 검색</translation>
     </message>
     <message>
         <location filename="../../libsearch/appsearch/app-search-plugin.cpp" line="154"/>
         <source>Application</source>
-        <translation>응용 프로그램</translation>
+        <translation>응용 프로그람</translation>
     </message>
     <message>
         <location filename="../../libsearch/appsearch/app-search-plugin.cpp" line="199"/>
         <source>Application Description:</source>
-        <translation>응용 프로그램 설명:</translation>
+        <translation>응용 프로그람 설명:</translation>
     </message>
 </context>
 <context>
@@ -173,12 +173,12 @@
     <message>
         <location filename="../../libsearch/searchinterface/searchtasks/app-search-task.cpp" line="55"/>
         <source>Application</source>
-        <translation>응용 프로그램</translation>
+        <translation>응용 프로그람</translation>
     </message>
     <message>
         <location filename="../../libsearch/searchinterface/searchtasks/app-search-task.cpp" line="60"/>
         <source>Application search.</source>
-        <translation>응용 프로그램 검색.</translation>
+        <translation>응용 프로그람 검색.</translation>
     </message>
 </context>
 <context>
@@ -400,7 +400,7 @@
         <location filename="../../libsearch/index/file-search-plugin.cpp" line="159"/>
         <location filename="../../libsearch/index/file-search-plugin.cpp" line="279"/>
         <source>Can not get a default application for opening %1.</source>
-        <translation>%1을(를) 여는 기본 응용 프로그램을 가져올 수 없습니다.</translation>
+        <translation>%1을(를) 여는 기본 응용 프로그람을 가져올 수 없습니다.</translation>
     </message>
     <message>
         <location filename="../../libsearch/index/file-search-plugin.cpp" line="187"/>
@@ -506,7 +506,7 @@
         <location filename="../../libsearch/notesearch/note-search-plugin.cpp" line="131"/>
         <source>Application</source>
         <translatorcomment>应用</translatorcomment>
-        <translation>응용 프로그램</translation>
+        <translation>응용 프로그람</translation>
     </message>
 </context>
 <context>
@@ -547,32 +547,32 @@
     <message>
         <location filename="../../libsearch/searchinterface/search-result-property-info.cpp" line="68"/>
         <source>application desktop file path</source>
-        <translation>응용 프로그램 데스크톱 파일 경로</translation>
+        <translation>응용 프로그람 바탕 화면 파일 경로</translation>
     </message>
     <message>
         <location filename="../../libsearch/searchinterface/search-result-property-info.cpp" line="74"/>
         <source>application local name</source>
-        <translation>응용 프로그램 로컬 이름</translation>
+        <translation>응용 프로그람 로컬 이름</translation>
     </message>
     <message>
         <location filename="../../libsearch/searchinterface/search-result-property-info.cpp" line="80"/>
         <source>application icon name</source>
-        <translation>응용 프로그램 아이콘 이름</translation>
+        <translation>응용 프로그람 아이콘 이름</translation>
     </message>
     <message>
         <location filename="../../libsearch/searchinterface/search-result-property-info.cpp" line="86"/>
         <source>application description</source>
-        <translation>응용 프로그램 설명</translation>
+        <translation>응용 프로그람 설명</translation>
     </message>
     <message>
         <location filename="../../libsearch/searchinterface/search-result-property-info.cpp" line="92"/>
         <source>is online application</source>
-        <translation>온라인 응용 프로그램임</translation>
+        <translation>온라인 응용 프로그람임</translation>
     </message>
     <message>
         <location filename="../../libsearch/searchinterface/search-result-property-info.cpp" line="98"/>
         <source>application package name</source>
-        <translation>응용 프로그램 패키지 이름</translation>
+        <translation>응용 프로그람 패키지 이름</translation>
     </message>
 </context>
 <context>
